lighthouse-core/test/report/html/renderer/util-test.js

```diff
@@ -189,6 +190,7 @@ describe('util helpers', () => {
       assert.equal(Util.getRootDomain(new URL('https://sub.example.com')), 'example.com');
       assert.equal(Util.getRootDomain(new URL('https://sub.example.tokyo.jp')), 'tokyo.jp');
       assert.equal(Util.getRootDomain(new URL('http://localhost')), 'localhost');
+      assert.equal(Util.getRootDomain(new URL('http://localhost:8080')), 'localhost');
```

Comment:
ahhhhh OK I gotcha now, nice sleuthing! :)

Reply:
"sleuthing" new word to my vocabulary 😄